packages/core/README.template

```diff
@@ -0,0 +1,28 @@
+# @tangleid/core
```

Comment:
Why `README.template` exists?

Reply:
We will use this `README.template` to generate the `README.md` in `packages/core`.
That `README.md` is used for the NPM package website page, e.g. [@tangleid/did - npm](https://www.npmjs.com/package/@tangleid/did). 😄

Reply:
Send another pull request for unify the use of `README.template`.

Reply:
Is that means that separate to another commit to add `README.md` and `README.template`? 🤔

Reply:
It depends on your progress to transit to npm packages.

Reply:
Unifying the use of README.template will not be done so fast, so I create an issue to track on it. 👌